wagtail_localize/templates/wagtail_localize/admin/confirm_convert_to_alias.html

```diff
@@ -0,0 +1,21 @@
+{% extends "wagtailadmin/base.html" %}
+{% load i18n wagtailadmin_tags %}
+{% block titletag %}{% blocktrans with title=page.get_admin_display_title %}Convert to alias {{ title }}{% endblocktrans %}{% endblock %}
+
+{% block content %}
+    {% trans "Convert to alias" as del_str %}
+    {% include "wagtailadmin/shared/header.html" with title=del_str subtitle=page.get_admin_display_title icon="doc-empty-inverse" %}
+
+    <div class="nice-padding">
+        <p>
+            {% trans 'Are you sure you want to convert this page into an alias?' %}
```

Comment:
How about something like:

| Are you sure you want to convert this page into an alias? This will overwrite the content of this page with the content of 'About' (English).

Included (English) as the source page locale in case the titles are the same